src/plugins/data_source_management/public/components/direct_query_data_sources_components/connection_detail/direct_query_connection_detail.tsx

```diff
@@ -322,7 +344,7 @@ export const DirectQueryDataConnectionDetail: React.FC<DirectQueryDataConnection
             <EuiFlexItem grow={false}>
               <EuiText className="overview-title">Description</EuiText>
               <EuiText size="s" className="overview-content">
-                {datasourceDetails.description || '-'}
+                {datasourceDetails.description || '\u2014'}
```

Comment:
so is this the change that no matter it is `useNewUX` enabled or not it will always show as the Em dash?

Reply:
yes